src/main/java/uk/gov/hmcts/darts/datamanagement/helper/StorageBlobFileDownloadHelper.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.darts.datamanagement.helper;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.common.datamanagement.api.DataManagementFacade;
+import uk.gov.hmcts.darts.common.datamanagement.component.impl.DownloadResponseMetaData;
+import uk.gov.hmcts.darts.common.datamanagement.component.impl.DownloadableExternalObjectDirectories;
+import uk.gov.hmcts.darts.common.entity.ExternalObjectDirectoryEntity;
+
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+public class StorageBlobFileDownloadHelper {
```

Comment:
What is the reason of an additional Helper layer?

Reply:
I added StorageBlobFileDownloadHelper to help with complexity of using the DataManagementFacade / DownloadableExternalObjectDirectories - so that I could mock the `DownloadResponseMetaData` for the tests in TranscriptionDownloaderTest / TranscriptionControllerDownloadTranscriptIntTest

Reply:
Removed in commits f3f8af7fc, b6caed8c6. Thanks